samples/bluetooth/peripheral_uart/sample.yaml

```diff
@@ -30,9 +30,8 @@ tests:
     extra_args: OVERLAY_CONFIG=prj_cdc.conf
     build_only: true
     platform_allow: nrf52840dk_nrf52840 nrf52833dk_nrf52833
-      nrf5340dk_nrf5340_cpuapp nrf5340dk_nrf5340_cpuapp_ns thingy53_nrf5340_cpuapp
+      nrf5340dk_nrf5340_cpuapp nrf5340dk_nrf5340_cpuapp_ns
```

Comment:
Why did you remove `thingy53_nrf5340_cpuapp` ? Wasn't cdc add for Thingy purpose ?

Reply:
Glad you asked. That was a mistake and should not be added. CDC is indeed enabled in Thingy:53 configurations. Thingy:53 uses 2 CDC_ACM instances, one for logging and one acting as UART backend for the sample purposes.

This overlay was added to test build configurations for other boards with async adapter. Thingy:53 has this enabled in the default configuration.